### PR TITLE
📦 NEW: Include new command called `health`.

### LINKED ISSUE #4: 📦 NEW: Include new command that support the health status on external services
This new command will be named as **health**, this one will get the following features:

- Show the current status on each services
- Show recent incidents

External Service Supported: 

- Github
- Trello

### PR DESCRIPTION
**Pull request by clouby**

### CODE REVIEW

File: src/api/generators/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const api = require('./api')
```
Comment (FlavioAPareja) [inline src/api/generators/index.js]:
```suggestion
exports.api = require('./api')
```

File: src/api/health/index.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        if (status.indicator !== 'none') {
+
+            const spinner = ora({
+                text: 'loading incidents history...',
+                prefixText: chalk.bold(`From ${name}`),
+                spinner: 'dots11',
+                color: 'yellow'
+            }).start()
+
+            const { items, ...rest } = await parser.parseURL(config.rss)
```
Comment (FlavioAPareja) [inline src/api/health/index.js]: Could you name this var using the type on it? 
something like `parserRSS`

File: src/api/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const Generators = require('./generators')
+const Health = require('./health')
+
+module.exports = {
+    Generators,
```
Comment (FlavioAPareja) [inline src/api/index.js]:
```suggestion
    Generators = require('generators'),
```
Same for health

File: src/api/health/index.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
+const fetch = require('node-fetch')
+const chalk = require('chalk')
+const figures = require('figures')
+const boxen = require('boxen')
+const link = require('terminal-link')
+const Parser = require('rss-parser')
+
+const { externalServices } = require('../../configs')
```
Comment (FlavioAPareja) [inline src/api/health/index.js:8]: Hey, you've this externalServices in the top of the document but the exports is in the bottom, just for semantic, could you move the export here?

**Review by FlavioAPareja — CHANGES_REQUESTED** [review]
Hey man, i really like your code, i learned a lot of modules and libraries for UX/DX, it let in the review some semantics requests 🤘🏼🤘🏼

**Comment by clouby** [comment]
@FlavioAandres Could you please check again ? :sweat_smile:

**Pull request merged**